### ISSUE TITLE
Ribbon toolbar = UX nightmare

### ISSUE THREAD

**Issue by LeadAssimilator** [microsoftfeedback/WinDbg-Feedback#133]
The ribbon based toolbar with few items in most ribbons is a complete waste and makes navigation and usage of the ui beyond difficult having to constantly switch ribbons.  The old windbg from the win 3.x ui era is much, much better with regard to its navigability and overall UX, despite being ancient in look.  Having to setup the quick access toolbars is the only way one can possibly cope with this hot mess of a UI, but even that is less than ideal and super tedious because none are defined out of the box.  Remember, the audience here is developers, not kids or grandma...everything should be readily available and in your face with as few clicks as possible to access - therefore:

The whole ribbon nonsense should all be replaced with command strips like VS with much smaller buttons.

**Comment by KenSykes** [comment]
(going through old feeback) Thanks for your feedback. We hope that in spite of your dislike of the ribbon UX you will find value in the other benefits of the new Windbg.

**Comment by LeadAssimilator** [comment]
@KenSykes No, absolutely not. Nothing is worth the UX pain that ribbons cause, not even ttd. Ribbons have no place in developer tools because they hide important functionality. Know your audience. If ribbons were a good UX choice, then VS would use them, but they don't, because they aren't! Classic windbg UI/UX, while ugly and stuck in win3.x era, is just way more usable day to day.

**Comment by LeadAssimilator** [comment]
Recreated as #289 because the problem of an unusable ux still exists.
